packages/template-retail-react-app/app/pages/checkout/partials/pickup-address.jsx

```diff
@@ -48,7 +48,10 @@ export default function PickupAddress() {
         city: store?.city,
         countryCode: store?.countryCode,
         postalCode: store?.postalCode,
-        stateCode: store?.stateCode
+        stateCode: store?.stateCode,
+        firstName: store?.name,
+        lastName: '',
```

Comment:
Following SFRA implementation and using the store name and a placeholder for the `lastName` when updating the shipping address.